eva.xsh

```diff
@@ -49,6 +49,9 @@ def main():
     tasks = get_changed_tasks(args)
     times = args.times
     for task in tasks:
+        run_file = os.path.join(config.baseline_path, '%s/run.xsh' % task)
+        if not os.path.exists(run_file):
+            continue
```

Comment:
Add a warning here, "skip xxx directory, it is not a task"

Reply:
ok~